deposit_contract.sol

```diff
@@ -110,7 +110,7 @@ contract DepositContract is IDepositContract {
             sha256(abi.encodePacked(amount, bytes24(0), signature_root))
         ));
         // Verify computed and expected deposit data roots match
-        require(node == deposit_data_root);
+        require(node == deposit_data_root, "DepositContract: reconstructed DepositData does not match supplied deposit_data_root");
```

Comment:
Maybe too long?

Reply:
I think the message just needs to be unique, you cannot grasp the full meaning without looking at the source anyway.